packages/functional-tests/tests/misc/relayIntegration.spec.ts

```diff
@@ -0,0 +1,139 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { FirefoxCommand } from '../../lib/channels';
+import { expect, test } from '../../lib/fixtures/standard';
+import { relayDesktopOAuthQueryParams } from '../../lib/query-params';
+import { getCode } from 'packages/fxa-settings/src/lib/totp';
+
+const AGE_21 = '21';
+
+test.describe('relay integration', () => {
+  test.beforeEach(async ({ pages: { configPage } }) => {
+    // Ensure that the feature flag is enabled
+    const config = await configPage.getConfig();
+    test.skip(
+      config.showReactApp.signUpRoutes !== true,
+      'Skip tests if not on React signUpRoutes'
```

Comment:
Not sure if I feel we need this, but I'm OK leaving it in :P (It does make me want to look at any test cleanup tickets we've got filed though heh)